completions/ant

```diff
@@ -1,5 +1,30 @@
 # bash completion for ant and phing                        -*- shell-script -*-
 
+_ant_parse_targets()
+{
+    local line basedir
+
+    basedir=$(dirname "$1")
```

Comment:
Wow, parameter expansion is a whole new world for me. What are the advantages of `${1%/*}` over `dirname`? Because I am afraid about its readability.